lua/aerial/backends/init.lua

```diff
@@ -161,9 +161,13 @@ M.set_symbols = function(backend_name, bufnr, items)
   local util = require("aerial.util")
   local window = require("aerial.window")
 
+  if config.post_add_all_symbols then
+    items = config.post_add_all_symbols(bufnr, items, ctx)
```

Comment:
Would be nice to add a nil-check here with a friendly error message in case the user mutates the symbols in-place and forgets to return them from the function.